components/v1/MarketDetailPage/ETHRISEPage.tsx

```diff
@@ -997,7 +997,7 @@ const ETHRISEPage: FunctionComponent<ETHRISEPageProps> = ({}) => {
                 </div>
             </div>
 
-            <div className="sm:hidden">
+            <div className="mt-20 sm:hidden">
```

Comment:
@math-marcellino I think we better to not use margin in this element. How about margin bottom in this div instead?

https://github.com/risedle/interface/blob/9425fa504e933711a132b08ab96db9b90640adb6/components/v1/MarketDetailPage/ETHRISEPage.tsx#L135

Reply:
Okay sure, let me fix it. Thank you for the review! @pyk